src/3rd_party/QNX_Boost.cmake

```diff
@@ -43,19 +43,21 @@ BOOTSTRAP
 
 if(${CMAKE_SYSTEM_PROCESSOR} MATCHES ".*aarch64")
   set(ADDRESS_MODEL "64")
+elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES ".*x86_64")  
+  set(ADDRESS_MODEL "32_64")
 else()  
   set(ADDRESS_MODEL "32")
 endif ()
 
-set(BOOST_BUILD_COMMAND ./b2 address-model=${ADDRESS_MODEL} cxxflags=-std=gnu++11 target-os=qnxnto toolset=gcc-nto${CMAKE_SYSTEM_PROCESSOR} define=__QNXNTO__)
+set(BOOST_BUILD_COMMAND ./b2 address-model=${ADDRESS_MODEL} cxxflags="-stdlib=libstdc++" linkflags="-stdlib=libstdc++" target-os=qnxnto toolset=gcc-nto${CMAKE_SYSTEM_PROCESSOR} define=__QNXNTO__)
 set(BOOST_INSTALL_COMMAND ${BOOST_BUILD_COMMAND} install > boost_install.log)
 if (${3RD_PARTY_INSTALL_PREFIX} MATCHES "/usr/local")
   set(BOOST_INSTALL_COMMAND sudo  ${BOOST_BUILD_COMMAND} install > boost_install.log)
 endif()
 include(ExternalProject)  
 ExternalProject_Add(
   Boost
-  URL https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz
+  URL http://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz
```

Comment:
@BSolonenko what is the reason of not secure pull?

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e